pinot-common/src/main/java/org/apache/pinot/common/controller/ControllerRequestClient.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.controller.helix;
+package org.apache.pinot.common.controller;
```

Comment:
```suggestion
package org.apache.pinot.common.utils.controller;
```

Reply:
Tried to follow the pattern, e.g. MinionClient is also under package `org.apache.pinot.common` https://github.com/apache/pinot/tree/master/pinot-common/src/main/java/org/apache/pinot/common/minion

Reply:
oh i see. in this case we should move `common.utils.http.HttpClient` to `common.http.HttpClient` as well. since the `common.http` package is already there. (sorry for missing this in previous PRs)

Reply:
I actually feel we should move `org.apache.pinot.common.http` to `org.apache.pinot.common.utils.http` :p